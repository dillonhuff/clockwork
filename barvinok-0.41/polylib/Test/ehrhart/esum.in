9 7
   1   0   0   0   0   1   0
   1   9   0   0   0  13 -22
   1  -9   0   0   0 -13  38
   1   0   0   0   1   0  -1
   1   1   0   0 -13   0  12
   1   0   0   1   0   0  -1
   1   1   0 -13   0   0  12
   1   0   1   0   0   0  -1
   1   1 -13   0   0   0  12
0 3
