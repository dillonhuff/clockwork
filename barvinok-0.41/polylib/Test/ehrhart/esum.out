---------------------------------------
Domain:
         P  -1 >= 0
         -13P + 29 >= 0

Vertices:
[ 1,  1,  1,  1 ]
[ -13P/9+38/9 ,  -P/9+146/117 ,  -P/9+146/117 ,  -P/9+146/117  ]
[ -13P/9+38/9 ,  1,  -P/9+146/117 ,  -P/9+146/117  ]
[ -13P/9+38/9 ,  -P/9+146/117 ,  1,  -P/9+146/117  ]
[ -13P/9+38/9 ,  1,  1,  -P/9+146/117  ]
[ -13P/9+38/9 ,  -P/9+146/117 ,  -P/9+146/117 ,  1 ]
[ -13P/9+38/9 ,  1,  -P/9+146/117 ,  1 ]
[ -13P/9+38/9 ,  -P/9+146/117 ,  1,  1 ]
[ -13P/9+38/9 ,  1,  1,  1 ]

Ehrhart Polynomial:
( [ 0, 1/78732, 1/157464, 0, 0, 0, 0, 0, 0 ]_P * P^4 + [ 0, -47/39366, -49/78732, 0, 0, 0, 0, 0, 0 ]_P * P^3 + [ 0, 1037/26244, 1133/52488, 0, 0, 0, 0, 0, 0 ]_P * P^2 + [ 0, -21197/39366, -24451/78732, 0, 0, 0, 0, 0, 0 ]_P * P + [ 0, 49210/19683, 30305/19683, 0, 0, 0, 0, 0, 0 ]_P )

---------------------------------------
Domain:
         - P + 1 >= 0
         P  >= 0

Vertices:
[ -13P/9+38/9 ,  -P/9+146/117 ,  -P/9+146/117 ,  -P/9+146/117  ]
[ -13P/9+38/9 ,  1,  -P/9+146/117 ,  -P/9+146/117  ]
[ -13P/9+38/9 ,  -P/9+146/117 ,  1,  -P/9+146/117  ]
[ -13P/9+38/9 ,  1,  1,  -P/9+146/117  ]
[ -13P/9+38/9 ,  -P/9+146/117 ,  -P/9+146/117 ,  1 ]
[ -13P/9+38/9 ,  1,  -P/9+146/117 ,  1 ]
[ -13P/9+38/9 ,  -P/9+146/117 ,  1,  1 ]
[ -13P/9+38/9 ,  1,  1,  1 ]
[ -13P/9+22/9 ,  -P/9+10/9 ,  -P/9+10/9 ,  -P/9+10/9  ]
[ -13P/9+22/9 ,  1,  -P/9+10/9 ,  -P/9+10/9  ]
[ -13P/9+22/9 ,  -P/9+10/9 ,  1,  -P/9+10/9  ]
[ -13P/9+22/9 ,  1,  1,  -P/9+10/9  ]
[ -13P/9+22/9 ,  -P/9+10/9 ,  -P/9+10/9 ,  1 ]
[ -13P/9+22/9 ,  1,  -P/9+10/9 ,  1 ]
[ -13P/9+22/9 ,  -P/9+10/9 ,  1,  1 ]
[ -13P/9+22/9 ,  1,  1,  1 ]

Ehrhart Polynomial:
( [ -2/729, -2/729, -2/729, -1/729, -2/729, -2/729, -2/729, -2/729, -2/729 ]_P * P^3 + [ 2/27, 20/243, 13/243, 1/81, 8/243, 10/243, 4/81, 14/243, 16/243 ]_P * P^2 + [ -2/3, -200/243, -125/243, -1/27, -32/243, -50/243, -8/27, -98/243, -128/243 ]_P * P + [ 2, 2000/729, 1339/729, 1/27, 128/729, 250/729, 16/27, 686/729, 1024/729 ]_P )

